### PR TITLE
feat: add code samples and tests for overlay creation

### PR DESCRIPTION
**Pull request by irataxy**
b:181782257

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
     "test": "c8 mocha build/test"
   },
   "dependencies": {
+    "express": "^4.17.1",
```
Comment (bcoe) [inline package.json:42]: I don't think we need this `express` dependency -- at least I don't see it being used.

File: samples/createJobWithAnimatedOverlay.js [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    const [response] = await transcoderServiceClient.createJob(request);
+    console.log(`Job: ${response.name}`);
+  }
+
+  createJobFromAnimatedOverlay();
+  // [END transcoder_create_job_with_animated_overlay]
+}
+
+// node createJobFromAnimatedOverlay.js <projectId> <location> <inputUri> <overlayImageUri> <outputUri>
+process.on('unhandledRejection', err => {
```
Comment (bcoe) [inline samples/createJobWithAnimatedOverlay.js:127]: awesome, glad to see the `unhandledRejection`.

File: samples/test/transcoder.test.js [hunk 3]
```diff
@@ -272,8 +275,102 @@ describe('Job functions adhoc', () => {
       `node getJobState.js ${projectId} ${location} ${this.adhocJobId}`,
       {cwd}
     );
-    // TODO(bcoe): remove this debug information once passing:
-    console.info(output.toString('utf8'));
+    assert.ok(output.includes('Job state: SUCCEEDED'));
+  });
+});
+
+describe('Job with static overlay functions', () => {
```
Comment (bcoe) [inline samples/test/transcoder.test.js:282]: tests look very solid 👍

File: samples/createJobWithStaticOverlay.js [hunk 4]
```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright 2021, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (bcoe) [inline samples/createJobWithStaticOverlay.js:3]: nit: we usually hyphen case our Node.js samples.

**Review by bcoe — CHANGES_REQUESTED** [review]
Just a small nit, we usually hyphen-case JavaScript files, so:

```
create-job-with-static-overlay.js
```

File: samples/createJobWithAnimatedOverlay.js (hunk 2 above)
Comment (irataxy) [reply samples/createJobWithAnimatedOverlay.js:127]: +1

File: package.json (hunk 1 above)
Comment (irataxy) [reply package.json]: Removed in https://github.com/googleapis/nodejs-video-transcoder/pull/39/commits/9d65c25a794f74992306219d6c1e3f53ee47c08b

File: samples/createJobWithStaticOverlay.js (hunk 4 above)
Comment (irataxy) [reply samples/createJobWithStaticOverlay.js:3]: Ack. Do I need to change this? I will then have to rename every .js file in the samples.

**Review by bcoe — APPROVED** [review]

**Pull request merged**